### DISCUSSION TITLE
Error when running ./run-docker.sh

### DISCUSSION THREAD

**Discussion by dongyanchu** [Xilinx/finn#816] [Q&A]
Hi, I met a problem when running run-docker.sh. I ran this command in ubuntu based on wsl.
Here is the information I got:

docker: Error response from daemon: pull access denied for xilinx/finn, repository does not exist or may require 'docker login': denied: requested access to the resource is denied.
See 'docker run --help'.

I ran the command "sudo docker login" and got the information "login succeeded".
But I cannot find the image "xilinx/finn" in Docker Hub.
Can someone help me?

**Comment by dongyanchu** [comment]
Here is the whole information I got:

Please set PLATFORM_REPO_PATHS pointing to Vitis platform files (DSAs).
This is required to be able to use Alveo PCIe cards.
fatal: not a git repository (or any parent up to mount point /mnt)
Stopping at filesystem boundary (GIT_DISCOVERY_ACROSS_FILESYSTEM not set).
WARNING: No blkio throttle.read_bps_device support
WARNING: No blkio throttle.write_bps_device support
WARNING: No blkio throttle.read_iops_device support
WARNING: No blkio throttle.write_iops_device support
Running container only
Docker container is named finn_dev_legion
Docker tag is named xilinx/finn : .xrt_202210.2.13.466_18.04-amd64-xrt
Mounting /tmp/finn_dev_legion into /tmp/finn_dev_legion
Mounting /opt/xilinx into /opt/xilinx
Port-forwarding for Jupyter 8888:8888
Port-forwarding for Netron 8081:8081
Vivado IP cache dir is at /tmp/finn_dev_legion/vivado_ip_cache
Using default PYNQ board Pynq-Z1
Successfully checked out qonnx at commit dd35a8ff49d7225a07ffceeebe25a6361df48349
Successfully checked out finn-experimental at commit 9cbd2787b5160e2b44e0e8164a0df1457dbd5366
Successfully checked out brevitas at commit a5b71d6de1389d3e7db898fef72e014842670f03
Successfully checked out pyverilator at commit 766e457465f5c0dd315490d7b9cc5d74f9a76f4f
Successfully checked out cnpy at commit 4e8810b1a8637695171ed346ce68f6984e585ef4
Successfully checked out finn-hlslib at commit d27f6b6c5d8f1bb208db395659389603f63ad4be
Successfully checked out oh-my-xilinx at commit d1065a788219ca0eb54d5e57600b1f9d7f67d4cc
Successfully checked out avnet-bdf at commit 2d49cfc25766f07792c0b314489f21fe916b639b
Successfully checked out xil-bdf at commit 8cf4bb674a919ac34e3d99d8d71a9e60af93d14e
Successfully checked out kv260-som-bdf at commit 98e0d3efc901f0b974006bc4370c2a7ad8856c79
Board files folder content mismatch, removing and re-downloading
Downloading and extracting board files...
ERROR: "docker buildx build" requires exactly 1 argument.
See 'docker buildx build --help'.

Usage:  docker buildx build [OPTIONS] PATH | URL | -

Start a build
Unable to find image 'xilinx/finn:latest' locally
docker: Error response from daemon: pull access denied for xilinx/finn, repository does not exist or may require 'docker login': denied: requested access to the resource is denied.
See 'docker run --help'.

**Reply by MingeCC** [reply]
兄弟，你这个问题解决了吗，怎么搞的

**Comment by wz19980621** [comment]
i have same problem.Do you solve it?

**Reply by prawin-kumar** [reply]
How were you able to resolve this @dongyanchu?

**Reply by MingeCC** [reply]
I have also encountered it. Have you resolved it?

**Comment by deadend10086** [comment]
soooooo sad anyone could tell me how to solve this problem？

**Comment by alperen-aksoy** [comment]
Did someone solve this issue?

**Comment by alperen-aksoy** [comment]
Worked for me when using v0.10.1 with `git checkout v0.10.1`

**Reply by deadend10086** [reply]
tbn, the FINN has not been updated for years.... I have changed to hls4ml & vitisAI.

**Reply by aksoyalpi** [reply]
I also tried hls4ml, but binary quantization with 1 bit for weights and activations is not supported at all
